### PR TITLE
regroup log entry fields

### PR DESCRIPTION
**Pull request by honnix**
@danielnorberg Does this make sense?

I removed those `@Nullable`, not sure how they could be `null`.

### CODE REVIEW

**Comment by honnix** [comment]
@ulzha PTAL as well.

File: contrib/flo-styx/src/main/java/com/spotify/flo/contrib/styx/StructuredLogMessage.java [hunk 1]
```diff
@@ -22,32 +22,38 @@
...
-  @JsonProperty @Nullable String styx_parameter();
-  @JsonProperty @Nullable String styx_execution_id();
-  @JsonProperty @Nullable String styx_trigger_id();
-  @JsonProperty @Nullable String styx_trigger_type();
-  @JsonProperty @Nullable String task_id();
-  @JsonProperty @Nullable String task_name();
-  @JsonProperty @Nullable String task_args();
+  @AutoMatter
+  interface Workflow {
+    @JsonProperty String styx_component_id();
```
Comment (danielnorberg) [inline contrib/flo-styx/src/main/java/com/spotify/flo/contrib/styx/StructuredLogMessage.java]: Would it make sense to structure things even more then?

```yaml
{
  "workflow": {
    "styx": {
      "workflow_id":"...",
      "component_id":"...",
      "workflow_id":"...",
      "docker_args":"...",
      "docker_image":"...",
      "commit_sha":"..."
    },
    "framework": "flo|luigi",
    "task": {
      "id": "...",
      "name": "...",
      "args": "..."
    }
  },
  "time": "...",
  "severity": "...",
  "logger": "...",
  "thread": "...",
  "message": "..."
}
```

Comment (honnix) [reply contrib/flo-styx/src/main/java/com/spotify/flo/contrib/styx/StructuredLogMessage.java]: Absolutely.

**Comment by danielnorberg** [comment]
@honnix Yeah, those fields can no longer be null (at least not as produced by this code, things might be different if we try to parse some log messages from the future etc).

**Comment by honnix** [comment]
It that ever happens, we can change it safely I think. So far just to get rid of those IntelliJ warnings about potential NPE, which are annoying.

**Comment by danielnorberg** [comment]
yolo!

**Pull request merged**